tamr_toolbox/low_latency/llm.py

```diff
@@ -0,0 +1,181 @@
+from typing import List, Dict, Optional, Literal
+import json
+from collections import defaultdict
```

Comment:
What is defaultdict ?

Reply:
It's a dict with the addition that you supply a default value when you initialize it. Then if you later try to access a key which doesn't exist, you get the default instead of a `KeyError`